vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -455,6 +465,18 @@ protected boolean allSidesAllocated() {
     return sides.size() == allocatedSideCount;
   }
 
+  /**
+   * @param side Name of a side to see if it's a "solo side"
+   * @return True if the side is "Solitaire", "Solo", "Moderator", or "Referee"
+   */
+  public static boolean isSoloSide(String side) {
+    return SOLITAIRE.equals(side) ||
+           SOLO.equals(side)      ||
+           MODERATOR.equals(side) ||
+           REFEREE.equals(side);
+  }
```

Comment:
This is going to be fun when someone is playing a Start Wars game... Han may need to shoot first...

Is there a better way we can do this than string matching?

Reply:
That's definitely something in the back of my mind. When I pulled this out to be a separate method I was definitely thinking about making it a single-centralized-method for handling this kind of check, and that we'd probably eventually want to "do a better job". I think the "right thing to do" is to extend the PlayerRoster to let Solo/Moderator player sides be explicitly named by the module designer w/ some kind of checkbox or something. Which seems more like a future PR?